src/UserEscrow.sol

```diff
@@ -35,7 +35,14 @@ contract UserEscrow is Auth {
     function transferOut(address token, address destination, address receiver, uint256 amount) external auth {
         require(destinations[token][destination] >= amount, "UserEscrow/transfer-failed");
         require(
-            receiver == destination || (ERC20Like(token).allowance(destination, receiver) >= amount),
+            /**
+             * @dev
+             * transferOut can only be initiated by the destination address or an authorized admin.
+             * The check is just an additional protection to secure destination funds in case of compromized auth.
+             * Since userEscrow is not able to decrease the allowance for the receiver,
+             * a transfer is only possible in case receiver has received the full allowance from destination address.
+             */
+            receiver == destination || (ERC20Like(token).allowance(destination, receiver) == type(uint256).max),
```

Comment:
To be consistent
```suggestion
             /// @dev transferOut can only be initiated by the destination address or an authorized admin.
             ///      The check is just an additional protection to secure destination funds in case of compromized auth.
             ///      Since userEscrow is not able to decrease the allowance for the receiver,
             ///      a transfer is only possible in case receiver has received the full allowance from destination address.
            receiver == destination || (ERC20Like(token).allowance(destination, receiver) == type(uint256).max),
```